smithy-model/src/main/java/software/amazon/smithy/model/transform/RenameShapes.java

```diff
@@ -61,6 +63,11 @@ Model transform(ModelTransformer transformer, Model model) {
                 .map(ShapeId::toString)
                 .collect(Collectors.toSet());
 
+        // TODO: this transform serializes the model, then deserializes it. Because of this, if the model
```

Comment:
This work seems done, maybe re-word the TODO to be clearer about not using serialization and move it to the top-level comment?